client-js/TUTORIAL.md

```diff
@@ -0,0 +1,202 @@
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Installing Dependencies
+
+Dependencies can be installed using the following command in the `client-js` directory:
+
+```js
+npm i
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs`.
+As every deploy created by the SDK you can send it using `.send(rpcUrl)` method providing rpc url that you want to use. It will return deployHash. 
+
+```js
+
+  const contractClient  = new CEP78Client(process.env.NODE_URL!, process.env.NETWORK_NAME!);
+
+  const installDeploy = await contractClient.install(
+    {
+      collectionName: "my-collection",
+      collectionSymbol: "MY-NFTS",
+      totalTokenSupply: "1000",
+      ownershipMode: NFTOwnershipMode.Transferable,
+      nftKind: NFTKind.Physical,
+      jsonSchema: {
+        properties: {
+          color: { name: "color", description: "", required: true },
+          size: { name: "size", description: "", required: true },
+          material: { name: "material", description: "", required: true },
+          condition: { name: "condition", description: "", required: false },
+        },
+      },
+      nftMetadataKind: NFTMetadataKind.CustomValidated,
+      identifierMode: NFTIdentifierMode.Ordinal,
+      metadataMutability: MetadataMutability.Immutable,
+      mintingMode: MintingMode.Installer,
+    },
+    "165000000000",
+    FAUCET_KEYS.publicKey,
+    [FAUCET_KEYS]
+  );
+
+  const hash = await installDeploy.send(process.env.NODE_URL!);
+
+```
+
+`InstallArgs` are specified sd follows:
+
+* `collectionName` - The name of the NFT collection, passed in as a `String`. **This parameter is required and cannot be changed post installation**.
+
+* `collectionSymbol` - The symbol representing a given NFT collection, passed in as a `String`. **This parameter is required and cannot be changed post installation**.
+
+* `totalTokenSupply` - The total number of NFTs that a specific contract instance will mint passed in as a `U64` value. **This parameter is required and cannot be changed post installation**.
+
+* `ownershipMode` - The `OwnershipMode` modality that dictates the ownership behavior of the NFT contract. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `nftKind` - The `NFTKind` modality that specifies the off-chain items represented by the on-chain NFT data. This argument is passed in as a `u8` value and is required at the time of installation.
+
+* `jsonSchema` - The JSON schema for the NFT tokens that will be minted by the NFT contract passed in as a `String`. This parameter is required if the metadata kind is set to `CustomValidated(4)` and **cannot be changed post installation**.
```

Comment:
Hi, I believe this parameter is also required for 0/1/2/3 metadata kind but it can be empty for Raw(3) only. CustomValidated(4) requires however a specific formatted schema.